src/rest-server/deploy/rest-server.yaml.template

```diff
@@ -33,45 +33,45 @@ spec:
       hostPID: false
       containers:
       - name: rest-server
-        image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}rest-server:{{ clusterinfo['dockerregistryinfo']['docker_tag'] }}
+        image: {{ cluster_cfg['cluster']['docker-registry']['prefix'] }}rest-server:{{ cluster_cfg['cluster']['docker-registry']['tag'] }}
         imagePullPolicy: Always
         env:
         - name: LAUNCHER_WEBSERVICE_URI
-          value: {{ clusterinfo['restserverinfo']['webservice_uri'] }}
+          value: {{ cluster_cfg['yarn-frameworklauncher']['webservice'] }}
         - name: HDFS_URI
-          value: {{ clusterinfo['restserverinfo']['hdfs_uri'] }}
+          value: http://{{ cluster_cfg['hadoop-name-node']['master-ip'] }}:9000
```

Comment:
maybe for the port hardcode problems, you could ask the service owner to make it as a new configuration

Reply:
@mzmssg any ideas?